corehq/apps/reports/filters/dates.py

```diff
@@ -44,6 +44,25 @@ def report_labels(self):
         })
 
 
+class HiddenLastMonthDateFilter(DatespanFilter):
+    """
+    A filter that returns last month as datespan
+    but is hidden since datespan should be fixed to last month
+    """
+    template = "reports/filters/month_datespan.html"
+    label = ugettext_lazy("Date Range")
+    slug = "datespan"
+    inclusive = True
+
+    @property
+    def datespan(self):
+        now = datetime.datetime.utcnow()
+        last_month = DateSpan.from_month(now.month - 1, now.year)
```

Comment:
I don't think this will work in January (the month will be zero).  You can use [`add_months`](https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/dates.py#L79-L88) to wrap around the year boundary:
```python
In [1]: from dimagi.utils.dates import add_months

In [2]: add_months(2016, 5, 1)
Out[2]: (2016, 6)

In [3]: add_months(2016, 5, -3)
Out[3]: (2016, 2)

In [4]: add_months(2016, 1, -1)
Out[4]: (2015, 12)
```
Something like
```python
year, month = add_months(now.year, now.month, -1)
last_month = DateSpan.from_month(month, year)
```